### DISCUSSION TITLE
Options for ASW, SSW, or motion smoothing

### DISCUSSION THREAD

**Discussion by tururuco** [alvr-org/ALVR#1395] [General]
ALVR settings for any of these would be great for those of us with lower end pcs that want a higher framerate but don't want to sacrifice an excessive amount of video quality. Synchronous warp (SSW) would be perfect for quest users because it could increase the framerate by using the quest 2s hardware to generate frames in between the ones generated by the computer.
(I have no programming experience or significant knowledge so I may be 100% wrong but I hope I'm not)

**Comment by zmerp** [comment]
There is already an issue about this. Indeed we could implement SSW (meaning only one frame is rendered every two and the second frame is interpolated with optical flow). One thing that we can implement sooner is just the framerate halving part, but letting the client use ATW to simply reproject the last available frame.

**Reply by elafynoh** [reply]
i would also be highly interested in having SSW in ALVR. I am about to switch to Linux for a couple of reasons and i use SSW in Virtual Dektop almost all the time for many games. Some games just do not work without it, especially on my RTX 3070 Laptop.

**Reply by daniellefisla** [reply]
I second this, this would be an amazing feature.
